terraform/app/modules/paas/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output docker_image_tag {
+  value = cloudfoundry_app.web_app.docker_image
```

Comment:
won't this return the full image name and not just the sha?
the deploy-pipeline is just expecting a sha?

Reply:
Thanks. I've made it clearer that `docker_image` consists of `repository:tag`